src/matlab2tikz.m

```diff
@@ -5624,29 +5634,23 @@ function printAll(m2t, env, fid)
     versionString = '';
 end
 % ==============================================================================
-function isHG2 = isHG2(m2t)
+function tf = isHG2(m2t)
 % Checks if graphics system is HG2 (true) or HG1 (false).
 % HG1 : MATLAB up to R2014a and currently all OCTAVE versions
 % HG2 : MATLAB starting from R2014b (version 8.4)
-    isHG2 = false;
-    if strcmpi(m2t.env,'MATLAB') && ...
-        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4])
-        isHG2 = true;
-    end
+    tf = strcmpi(m2t.env,'MATLAB') && ...
+        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4]);
 end
 % ==============================================================================
-function isBelow = isVersionBelow(env, versionA, versionB)
+function tf = isVersionBelow(env, versionA, versionB)
```

Comment:
I agree with most of your points but I prefer to concentrate the verbosity on the question being asked by the `is*` function, so that the naming is clear on what the function is doing. At the end we get 0 or 1 in all cases (with those functions), so I do not see much gain in coming up with a different output name for every function. This means that I am indifferent between `bool` and `tf` but see the `isValid` alternative as a good name for a function rather than for an output name. 

Anyways, that commit should be irrelevant to the PR, so we can always revert it.